### DISCUSSION TITLE
How to access functions via GET

### DISCUSSION THREAD

**Discussion by alrik11es** [appwrite/appwrite#3185] [Q&A]
I've created a PHP function and I can call this function using the Appwrite API without any problems.

I'd like to expose this function to internet and access via GET endpoint like http://myserver.com/function/xxxx without authentication.
How can I do this? I mean easily without creating a proxy-like front-end.

Don't know if it's possible but it would be nice to have an option in the functions to do this from settings.

**Comment by devmike123** [comment]


**Reply by alan1420** [reply]
Wrong answer.

**Comment by stnguyen90** [comment]
@alrik11es, you'll need some proxy/gateway to translate the request into the format Appwrite expects. You can try to use this: https://github.com/Meldiron/railway-webhook-proxy
